add Development Setup in docs

RubenHalman opened the pull request:
This PR adds detailed instructions for setting up a development environment for OrgCheck, following a package-based development model. The guide outlines the steps required to deploy OrgCheck as an unlocked package and provides clarity on namespace configuration, package creation, versioning, and deployment to scratch orgs.

RubenHalman commented:
@VinceFINET FYI

VinceFINET commented on README.md:
```diff
@@ -37,6 +37,9 @@ You can log any issues and new ideas in our tracker: https://github.com/Salesfor
 
+## Development Setup
+
+To set up a development environment and deploy OrgCheck as your own unlocked package, follow the steps outlined in the [Development Setup Guide](docs/development.md).
```
Please use "**Org Check**" instead of "**OrgCheck**". This is a ask from internal PR at salesforce to not use a "new" word that might be under an existing ownership. instead they told me to just put two words separated by a space.

VinceFINET commented on docs/development.md:
```diff
@@ -0,0 +1,100 @@
+<h1>Development Setup Guide</h1>
+
+<p>This guide will walk you through setting up a developer environment for OrgCheck, allowing you to deploy OrgCheck as your own unlocked package.</p>
```
use "Org Check"

VinceFINET commented on docs/development.md:
```diff
@@ -0,0 +1,100 @@
+    <li>Salesforce CLI installed</li>
+</ul>
+
+<h2>Step 1: Clone the OrgCheck Project</h2>
```
use "Org Check"

VinceFINET commented on docs/development.md:
```diff
@@ -0,0 +1,100 @@
+</ul>
+
+<h2>Step 1: Clone the OrgCheck Project</h2>
+<p>First, clone the OrgCheck project repository from GitHub:</p>
```
use "Org Check"

VinceFINET commented on docs/development.md:
```diff
@@ -0,0 +1,100 @@
+<h1>Development Setup Guide</h1>
```
i would prefer this md to use markdown syntaxt so that it can be included in the website as well!!
is it possible @RubenHalman ?

VinceFINET reviewed (commented):
THANK YOU that's a wonderfull idea you had!
Please see comment in the md.

RubenHalman commented:
@VinceFINET 
I have completed your feedback.
Additionally I make some changes to main readme:
- combining all links with respective text
- added socials links to more about
- fixed the reference to the License file by renaming the License file to License.MD
- removed the license text and added the reference in the MIT Badge Icon

These can be updated or removed as you see fit!

VinceFINET reviewed (approved):
Perfect!
